include/constants/battle.h

```diff
@@ -475,4 +475,7 @@
 // For the second argument of GetMoveTarget, when no target override is needed
 #define NO_TARGET_OVERRIDE 0
 
+// Makes sure gWrappedStringIds and sTrappingMoves have the same size
+#define TRAPPING_MOVES_COUNT 9
```

Comment:
instead of this define, please define B_MSG_WRAPPED_BIND, etc. for each gWrappedStringIds entry in `include/constants/battle_message.h`, with the final entry being TRAPPED_MOVES_COUNT. This will make it easier for users to add new trap moves. Then, you can use those defines as initializers in each array below.